docs/governance/working-groups/recommendations/policies-recommendations.md

```diff
@@ -0,0 +1,24 @@
+# Policies & Recommendations
+
+There are two types of guidliness that will be issued to govern the CCC [WG]s.
+
+## Policies
+
+Policies are guidelines which have been proposed by the community structure [WG] and which have been approved by [SC].
+
+All [WG]s must adhere to the policies.
+
+## Recommendations
+
+Recommendations are guidelines which have been suggested by the community structure [WG]. While these guidelines have not been approved by the [SC] and made into a policy it is strongly suggested that each [WG] follow the recommendations.
+
+## Upgrading a Recommendation to become a Policy
+
+In order for a recommendation to become a policy a [SC] sponsor must be found. The proposer can raise a PR with the necessary changes i.e. moving the reccomendation to the policies directory. The [SC] sponsor can call a [SC] vote and if approved by the majority the PR can be merged and the recommendation is now a policy.
```

Comment:
This could probably be worded a bit better. We didn't have a great night with our son as he isn't well so I am not at my peak today.